CppCoreGuidelines.md

```diff
@@ -17696,7 +17696,50 @@ Type erasure incurs an extra level of indirection by hiding type information beh
 
 ##### Example
 
-    ???
+    #include <memory>
+
+    class Object
+    {
+    public:
+        template <typename T>
+        Object(T && obj)
+            : concept_(std::make_unique<Model<T>>(std::forward<T>(obj))) {}
+
+        int getId() const { return concept_->getId(); }
```

Comment:
I think getters and setters have `get_id()` forms in other examples
such as 
* [C.131](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c131-avoid-trivial-getters-and-setters)
* [es50](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es50-dont-cast-away-const)